src/models/AmountMoney.ts

```diff
@@ -0,0 +1,7 @@
+interface AmountMoney {
+    amount?: number;
+    
+    currencyCode: string;
```

Comment:
`currencyCode` should be optional [per documentation](https://github.com/square/react-native-square-reader-sdk/blob/master/docs/reference.md#money).